.github/workflows/instrumentation-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Instrumentation tests
+
+on: [push, pull_request]
+
+jobs:
+  instrumentation-test:
```

Comment:
I suggest moving this job into the existing [test workflow](https://github.com/customerio/customerio-android/blob/main/.github/workflows/test.yml) to keep all tests together.

Reply:
I thought about this at first, but instrumentation tests take a lot longer than unit tests, and at times you have to run the test again because the AVD didn't run or was unknown other problems can happen with tests that involve an emulator running on the fly. 

If we tied both of them together, will they slow us down? we would need to re-run both tests on failure and also, will have to probably wait for long to know even if the problem was with unit tests etc.

Reply:
Jobs run in parallel by default and that is what I am suggesting - moving the entire job into the test workflow file. 

This will unit tests and instrumentation tests run in parallel and will allow you to re-run failed tests, only. You do not need to re-run both.